datasets/nwm-archive.yaml

```diff
@@ -84,15 +84,16 @@ DataAtWork:
   Tutorials:
     - Title: Explore the National Water Model Reanalysis in Zarr
       URL: https://nbviewer.jupyter.org/gist/rsignell-usgs/d3dfaf3cd3d8b39894a69b22127dfe38
-      AuthorName: Rich Signell
+      AuthorName: '[Rich Signell](https://www.usgs.gov/staff-profiles/richard-p-signell?qt-staff_profile_science_products=0#qt-staff_profile_science_products)'

```

Comment:
Hi @Patrick-Keown can you please just use the `AuthorURL` field for the urls rather than putting it in the `AuthorName` field as well? Thanks!

Reply:
For sure. Changed!

Reply:
You keep finding the edges of what we built. :) The `AuthorURL` field only supports one URL. So I'd say if there isn't one URL that can apply to all authors, you just drop that field altogether. We may add the ability to add multiple author urls, but unfortunately not yet.